### PR TITLE
fix the account,wait an abstract class

### PR DESCRIPTION
**Pull request by ReinaldoAguilar**

### CODE REVIEW

File: src/main/java/org/fundacionjala/pivotal/cucumber/selenium/pages/accounts/Accounts.java [hunk 1]
```diff
@@ -147,19 +149,19 @@ private ManageAccount clickManageAccount(final WebElement manageAccountButton) {
      *
      * @param webElementNameAccount webElement param
      * @param accountName           second param
-     * @return boolean is succesfull
+     * @return String is successful
      */
-    public boolean findAccount(final List<WebElement> webElementNameAccount, final String accountName) {
```
Comment (daniel-m10) [inline src/main/java/org/fundacionjala/pivotal/cucumber/selenium/pages/accounts/Accounts.java]: This method could be improved using lambdas.

File: src/main/java/org/fundacionjala/pivotal/cucumber/selenium/pages/accounts/Accounts.java [hunk 2]
```diff
@@ -147,19 +149,19 @@ private ManageAccount clickManageAccount(final WebElement manageAccountButton) {
...
-            if (text.equalsIgnoreCase(accountName)) {
-                return true;
+            if (accountName.equalsIgnoreCase(text)) {
+                return accountName;
             }
 
         }
-        return false;
+        return NOTEXITACCOUNT;
     }
```
Comment (daniel-m10) [inline src/main/java/org/fundacionjala/pivotal/cucumber/selenium/pages/accounts/Accounts.java]: You could improve this method.

File: src/main/java/org/fundacionjala/pivotal/cucumber/selenium/pages/accounts/SettingAccount.java [hunk 3]
```diff
@@ -18,7 +18,7 @@
      * This method doing click on "Ok alert Message" button.
```
Comment (pablopzsj) [inline src/main/java/org/fundacionjala/pivotal/cucumber/selenium/pages/accounts/SettingAccount.java]: The comment should be "This method is clicking on the "Ok alert Message" button."

File: src/main/java/org/fundacionjala/pivotal/cucumber/selenium/pages/menu/ToolBars.java [hunk 4]
```diff
@@ -66,23 +65,24 @@
...
-            DriverManager.getInstance().getWait().withTimeout(Environment.getInstance().getTimeout(), TimeUnit.SECONDS);
+            driver.manage().timeouts().implicitlyWait(Environment.getInstance().getTimeout(), TimeUnit.SECONDS);
+            wait.withTimeout(Environment.getInstance().getTimeout(), TimeUnit.SECONDS);
         }
         return userLogged;
     }
 
+
     /**
      * Whent the user is logged.
```
Comment (pablopzsj) [inline src/main/java/org/fundacionjala/pivotal/cucumber/selenium/pages/menu/ToolBars.java]: The comment should be "When the user is logged."

File: user.properties [hunk 5]
```diff
@@ -1,16 +1,16 @@
...
+browser=chrome
 timeout=15
 # End Configuration for Selenium
 # Configuration for Rest-assured
 # optional proxy IP = 172.31.90.146 / PORT = 3128
 proxyIp=
 proxyPort=
 baseURI=https://www.pivotaltracker.com/services/v5
-apiToken=
+apiToken=e41bb56cd02e58cf25da9a5a558e625a
```
Comment (rubercuellar) [inline user.properties]: Clean these values

**Pull request merged**